docs/velocity/admin/reference/configuration.md

```diff
@@ -18,10 +18,9 @@ There are a few "special" data types in the Velocity configuration.
 
 ### Chat
 
-Chat messages may be provided in legacy color code format or in JSON format.
+Chat messages may be provided in [MiniMessage](https://docs.adventure.kyori.net/minimessage/format.html) format.
```

Comment:
Update the adventure docs link. They changed to `docs.advntr.dev` (or smth close to that).